src/status/absent.py

```diff
@@ -0,0 +1,55 @@
+from src.decorators import event_listener
+from src.containers import UserDict
+from src.functions import get_players
+from src.messages import messages
+from src import channels
+
+__all__ = ["add_absent"]
+
+ABSENT = UserDict() # type: UserDict[users.User, Tuple[str, bool]]
+
+def add_absent(var, target, key, *, private=True):
+    if target not in get_players():
+        return
+
+    ABSENT[target] = (key, private)
+
+    for votee, voters in list(var.VOTES.items()):
+        if target in voters:
+            voters.remove(target)
+            if not voters:
+                del var.VOTES[votee]
+            break
+
+    from src.wolfgame import chk_decision, chk_win
+    chk_decision()
+    chk_win()
```

Comment:
Status modules should never have side effects like phase transitions. Move these out of add_absent and handle them by the code which calls them if necessary (priest and gunner are the only two I can think of which would need to).